projects/kit/components/text-area/text-area.style.less

```diff
@@ -19,15 +16,6 @@
 
     &[data-size='m'] {
         font: var(--tui-font-text-s);
-        min-height: (@height - 1.25rem);
```

Comment:
You could still keep it in LESS with no JS, just update value of the variable, i.e.:
```css
&[data-size='m'] {
  font: var(--tui-font-text-s);

  --tui-textarea-height: (@height - 1.25rem);
}
```

Reply:
cool, fixed